vm/src/vm.h

```diff
@@ -80,6 +83,114 @@ size_t moshell_vm_next_page(moshell_vm vm);
  */
 void moshell_vm_free(moshell_vm vm);
 
+/**
+ * A moshell value, either an i64, a unsigned byte, a double
+ * or an address
+ * */
+typedef union {
+    int64_t i;
+    uint8_t b;
+    double d;
+    const void *ptr;
```

Comment:
```suggestion
    void const *ptr;
```
The pointer is `const`, not the value behind.

Reply:
both